hashtable/LL_collision_resolution.js

```diff
@@ -43,7 +50,13 @@ class HashTable {
      * DJB2 Hash, 32-bit
      */
     djb2(key){
-        //your code here
+        let total = 0
```

Comment:
Where did you find this implementation of the djb2? When I looked it up I found it as 
`let hash = 5381; 
         for(const c of key){
             hash = (hash * 33) + c.charCodeAt(0)
            }
            return hash;`